brickflow/cli/projects.py

```diff
@@ -40,7 +40,9 @@
 class BrickflowProject(BaseModel):
     name: str
     path_from_repo_root_to_project_root: str = "."  # used for mono repo
-    path_project_root_to_workflows_dir: str  # used for repos with multiple batches of workflows
+    path_project_root_to_workflows_dir: (
```

Comment:
What was the reason for this change?
```suggestion
    path_project_root_to_workflows_dir: str
```

Reply:
It changed when I ran make fmt command.